### PR TITLE
Fix for Azure Application Gateway. Golden metrics and tags for AWS types

### PR DESCRIPTION
**Pull request by helenapm**
Hello Entity Platform, this is Helena :) 

AWSAUTOSCALINGGROUP
definition.yml: Adding region as key tag.

AWSCLOUDFRONTDISTRIBUTION
definition.yml: Adding region as key tag.
golden_metrics.yml: Adding uploaded and downloaded bytes as golden metrics

AWSDYNAMODBREGION
definition.yml: Adding key tags and golden metrics file
golden_metrics.yml: Adding system errors as golden metric

AWSDYNAMODBTABLE
definition.yml: Adding region and table status as key tags.
golden_metrics.yml: Fixing aggregation functions

AWSDYNAMODBGLOBALSECONDARYINDEX
definition.yml: Adding key tags and golden metrics file
golden_metrics: Provisioned and consumed read and write capacity units

AZUREAPPLICATIONGATEWAY
golden_metrics.yml: Fixing metric names and aggregation functions

### Checklist

* [X] I've read the guidelines and understand the acceptance criteria.
* [ ] The value of the attribute marked as `identifier` will be unique and valid. 
* [ ] I've confirmed that my entity type wasn't already defined. If it is I'm providing an
 explanation above.

### CODE REVIEW

File: definitions/infra-awscloudfrontdistribution/golden_metrics.yml [hunk 1]
```diff
@@ -14,3 +14,19 @@ totalErrorRate:
     where: provider='CloudFrontDistribution'
     facet: entityName
     eventId: entityGuid
+bytesUploaded:
```
Comment (IreneMLC) [inline definitions/infra-awscloudfrontdistribution/golden_metrics.yml:17]: this map key is duplicated and it causes the checks to fail.

File: definitions/infra-awscloudfrontdistribution/golden_metrics.yml [hunk 2]
```diff
@@ -14,3 +14,19 @@ totalErrorRate:
...
+  title: Uploaded bytes
+  query:
+    select: sum(provider.bytesUploaded.Sum)
+    from: LoadBalancerSample
+    where: provider='CloudFrontDistribution'
+    facet: entityName
+    eventId: entityGuid
+bytesUploaded:
```
Comment (IreneMLC) [inline definitions/infra-awscloudfrontdistribution/golden_metrics.yml:25]:
```suggestion
bytesDownloaded:
```

**Review by IreneMLC — APPROVED** [review]
Small naming issue aside look good!

File: definitions/infra-awscloudfrontdistribution/golden_metrics.yml (hunk 1 above)
Comment (helenapm) [reply definitions/infra-awscloudfrontdistribution/golden_metrics.yml:17]: Sorry, thanks for noticing @IreneMLC

**Review by naxhh — APPROVED** [review]

**Pull request merged**